librz/core/disasm.c

```diff
@@ -1130,6 +1130,7 @@ static void ds_build_op_str(RzDisasmState *ds, bool print_color) {
 
 		rz_parse_filter(core->parser, ds->vat, core->flags, ds->hint, source,
 			ds->str, sizeof(ds->str), core->print->big_endian);
+		RZ_FREE(source);
```

Comment:
I see `source` might be ds->opstr or the result of `rz_asm_op_get_asm`. Is it always right to free it?

Reply:
Yes, a copy of `source` is made anyway, and it is not used afterwards 

https://github.com/rizinorg/rizin/blob/5f4bc5ff59474f7425a25f6b464888abc89951c7/librz/core/disasm.c#L1117-L1133